.circleci/config.yml

```
@@ -300,15 +313,15 @@ jobs:
       - run: yarn grunt prepareDeploy
         # Install dependencies for Firebase functions to prevent parsing errors during deployment
         # See https://github.com/angular/angular.js/pull/16453
-      - run: yarn -cwd $PROJECT_ROOT/scripts/docs.angularjs.org-firebase/functions
+      - run: yarn -cwd ~/ng/scripts/docs.angularjs.org-firebase/functions
```

Wouldn't the following also work (since `working_directory` is set to `~/ng` for the executor)?

```suggestion
      - run: yarn -cwd scripts/docs.angularjs.org-firebase/functions
```

---

Good idea!